nemo/collections/asr/parts/utils/speaker_utils.py

```diff
@@ -395,8 +394,25 @@ def write_rttm2manifest(paths2audio_files, paths2rttm_files, manifest_file):
 
 
 def segments_manifest_to_subsegments_manifest(
-    segments_manifest_file: str, sub_segments_manifest_file: str = None, window: float = 1.5, shift: float = 0.75
+    segments_manifest_file: str,
+    sub_segments_manifest_file: str = None,
+    window: float = 1.5,
+    shift: float = 0.75,
+    min_subsegement_duration: float = 0.05,
 ):
+    """
+    Generate subsegments manifest from segments manifest file
+    Args
+    input:
+        segments_manifest file (str): path to segments manifest file, typically from VAD output
+        sub_segments_manifest_file (str): path to output sub segments manifest file (default (None) : writes to current working directory)
+        window (float): window length for segments to sub segments division
+        shift (float): hop length for sub segments shift 
+        min_subsegments_duration (float): exclude sub segments smaller than this duration value
```

Comment:
This is very minor point, but it's better to be no space between "sub" and "segment"
https://www.dictionary.com/browse/subsegment

"subsegment"
This is for the sake of supporting search of the specific word.